web/src/billing/upgrade.ts

```diff
@@ -14,6 +14,11 @@ const prices: Prices = {
 
 const ls = localstorage();
 const ls_selected_schedule = ls.get("selected_schedule");
+const ls_remote_server_plan_start_date = ls.get("remote_server_plan_start_date");
+let remote_server_plan_start_date: string =
+    typeof ls_remote_server_plan_start_date === "string"
+        ? ls_remote_server_plan_start_date
+        : "billing_cycle_end_date";
```

Comment:
Hmm when would we ever want to display the string `billing_cycle_end_date` in this fallback code?

Reply:
The is just setting default value for the plan start date dropdown in case there is no value found in the localstorage.

Reply:
I see -- I understand now (and in fact this value gets sent to the server to request that to be used as the date, right?).

Reply:
Yeah.